teuthology/__init__.py

```diff
@@ -14,10 +14,17 @@
 
 import logging
 import os
+import subprocess
 
+mydir = os.path.dirname(os.path.realpath(__file__))
 
-__version__ = '0.1.0'
-
+try:
+    __version__ = subprocess.check_output(
```

Comment:
I wonder if we should use strings like ``0.1.0-$hash``; just in case we ever do real releases...